src/juliaexepath.ts

```diff
@@ -13,7 +13,11 @@ let actualJuliaExePath: string = null
 async function setNewJuliaExePath(newPath: string) {
     actualJuliaExePath = newPath
 
-    child_process.exec(`"${newPath}" --version`, (error, stdout, stderr) => {
+    const env = {
+        JULIA_VSCODE_WORKSPACE: vscode.workspace.workspaceFolders[0].uri.fsPath
+    }
+
+    child_process.exec(`"${newPath}" --version`, { env: env }, (error, stdout, stderr) => {
```

Comment:
What is this value used for? Does it expect the Julia version for LS or the version for the REPL process? I tried to trace it and it seems like it is only used for telemetry? So I guess you expect this to be the version of julia that LS runs on?

Reply:
I think the main reason for this is to check whether the path is correct; getting the version is just a side effect.